Core/Utilities/DIRACError.py

```diff
@@ -0,0 +1,207 @@
+""" :mod: DIRACError
```

Comment:
I would suggest to call it DErrno rather than DIRACError as it is confusing to have DError and DIRACError